packages/ngtools/webpack/src/compiler_host.ts

```diff
@@ -28,24 +28,25 @@ const dev = Math.floor(Math.random() * 10000);
 
 export class WebpackCompilerHost implements ts.CompilerHost {
   private _syncHost: virtualFs.SyncDelegateHost;
+  private _memoryHost: virtualFs.SyncDelegateHost;
   private _changedFiles = new Set<string>();
   private _basePath: Path;
   private _resourceLoader?: WebpackResourceLoader;
+  private _sourceFileCache = new Map<string, ts.SourceFile>();
 
   constructor(
     private _options: ts.CompilerOptions,
     basePath: string,
     host: virtualFs.Host,
   ) {
-    this._syncHost = new virtualFs.SyncDelegateHost(new virtualFs.CordHost(host));
+    this._syncHost = new virtualFs.SyncDelegateHost(host);
+    this._memoryHost = new virtualFs.SyncDelegateHost(new virtualFs.SimpleMemoryHost());
     this._basePath = normalize(basePath);
   }
 
   private get virtualFiles(): Path[] {
-    return (this._syncHost.delegate as virtualFs.CordHost)
-      .records()
-      .filter(record => record.kind === 'create')
-      .map((record: virtualFs.CordHostCreate) => record.path);
+    return [...(this._memoryHost.delegate as unknown as { _cache: Map<Path, {}> })
```

Comment:
Can’t use unknown yet before google upgrades to TS3

Reply:
Done.